keyboards/model_f_labs/f62/keymaps/via/rules.mk

```diff
@@ -0,0 +1,5 @@
+VIA_ENABLE = yes
+LTO_ENABLE = yes
+BOOTMAGIC_ENABLE = no
```

Comment:
bootmagic gets force enabled if via is enabled, setting this to "no" isn't enough to disable bootmagic. 

If you absolutely need to disable bootmagic... adding this is the best way: 
```c
void bootmagic_scan(void) {}
```

Reply:
Updated the VIA keymap to disable bootmagic.